src/main/java/com/lhd/earlybirdapi/githubrepo/GithubRepoService.java

```diff
@@ -82,59 +86,20 @@ private Instant findLatestIssueCreatedAtTimestamp(String githubRepoId) {
     return Optional.of(issues[0]);
   }
 
-  // TODO: debug why this request wouldn't work with RestTemplate or the like
-  // this really should be an http request, not a cURL executed with ProcessBuilder
   private IssueDto[] postForGithubRepoIssues(String githubRepoId) {
+    String URL = "/repos/" + githubRepoId + "/issues";
     IssueDto[] issues;
 
     // TODO: each of the methods that could throw IOExceptions should handle it themselves
     // we could remove the try/catch here in that case, and create more specific custom exceptions
     try {
-      Process curlRequestProcess = executeCurlRequestAndRetrieveProcess(githubRepoId);
-      BufferedReader bufferedReader = getBufferedReaderFromProcess(curlRequestProcess);
-      String serializedResponse = getSerializedResponse(bufferedReader);
-      issues = new ObjectMapper().readValue(serializedResponse, IssueDto[].class);
-    } catch (IOException e) {
+      ResponseEntity<String> responseEntity = restTemplate.getForEntity(URL, String.class);
```

Comment:
do we want to capture this as a string? should we capture as a list of the object type we expect?